README.md

```diff
@@ -1,47 +1,185 @@
 # react-e [![CircleCI](https://circleci.com/gh/Originate/react-e.svg?style=svg)](https://circleci.com/gh/Originate/react-e)
 
-React helper to generate elements
+JSX replacement that doesn't need to be compiled.
 
-## Installing
-`npm install --save react-e`
+## Basic Usage
 
-## Example Usage
+```javascript
+e = require('react-e')
 
-```coffeescript
-e = require 'react-e'
-
-e '.container',
-  e 'button.primary', 'Click me'
-  e 'span.text', className: active: true, 'Some text'
+e(Modal,
+  e('.container',
```

Comment:
I still vote for using `$` instead of `e` here. It is a lot more intuitive than `e`. $ is in no way owned by jQuery, it's just a placeholder for "the DOM library you use".